olive/data/component/post_process_data.py

```diff
@@ -30,3 +30,22 @@ def text_classification_post_process(_output_data, **kwargs):
         _, preds = torch.max(_output_data, dim=1)
 
     return preds
+
+
+@Registry.register_post_process()
+def ner_post_process(_output_data, **kwargs):
```

Comment:
Any documentations need to be updated about how to use this?

Reply:
https://github.com/microsoft/Olive/pull/401/files#diff-fce744f2268555668fc1868242a7833b6691a78bc883d9dbfc50fcc23f70a0b9

Here I add the template config file for `ner` task. This function will not be exposed to user directly now. When user set the task as ner, the corresponding pre_process, post_process will be picked.

Reply:
https://github.com/microsoft/Olive/blob/main/docs/architecture.md#data-container Here is the doc to use that function internally now.